### PR TITLE
#1823: Cover GetAssetByIdInterface with integration test

### LINKED ISSUE #1823: Cover GetAssetByIdInterface with integration test
Cover `\Magento\AdobeStockAssetApi\Api\GetAssetByIdInterface` with integration test

### PR DESCRIPTION
**Pull request by yolouiese**
### Description (*)
This PR covers the integration test for GetAssetByIdInterface service

### Fixed Issues (if relevant)
1. Fixes magento/adobe-stock-integration#1823: Cover GetAssetByIdInterface with integration test

### CODE REVIEW

**Comment by yolouiese** [comment]
@magento run all tests

File: AdobeStockAsset/Test/Integration/Model/GetAssetByIdTest.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    /**
+     * @var GetAssetByIdInterface
+     */
+    private $getAssetById;
+
+    protected function setUp(): void
+    {
+        Bootstrap::getObjectManager()->configure([
+            'preferences' => [
+                GetAssetListInterface::class => GetAssetListMock::class
```
Comment (sivaschenko) [inline AdobeStockAsset/Test/Integration/Model/GetAssetByIdTest.php]: Please reuse the existing client mock instead of `GetAssetListMock`

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for PR @yolouiese ! Please see my comment

File: AdobeStockAsset/Test/Integration/Model/GetAssetByIdTest.php (hunk 1 above)
Comment (yolouiese) [reply AdobeStockAsset/Test/Integration/Model/GetAssetByIdTest.php]: Hi @sivaschenko, okay I'll just wait for https://github.com/magento/adobe-stock-integration/pull/1803 to be merged. Thank you.

Comment (sivaschenko) [reply AdobeStockAsset/Test/Integration/Model/GetAssetByIdTest.php]: @yolouiese it's merged now. Thanks!

**Comment by yolouiese** [comment]
@magento run all tests

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**